src/riscv/lib/src/jit/builder/control_flow_graph/tests.rs

```diff
@@ -55,6 +55,10 @@ struct TestInstrOutcome {
 
```

Comment:
nit: this is more of a `feat` type PR no? as it would actually make a difference to the performance of step counting